docs/run/fair-share-queue.mdx

```diff
@@ -60,4 +60,4 @@ As described above, all jobs submitted to the scheduler through the same project
 
 A wait-time estimate is provided through IBM Quantum Platform and via Qiskit. The computed time is the result of a scheduling simulation that predicts one possible execution pattern, given the current fair-share ordering of all the jobs waiting for that system and the approximate runtime of each job. The dynamic nature of the fair-share algorithm means that this estimated time is not fixed and can vary, sometimes dramatically. This wait time is also subject to limitations inherent in estimating the execution time for Qiskit Runtime jobs. For these jobs, where an accurate estimation of time is not feasible, the maximum allowed runtime is used as a proxy. In practice, this means that the duration for a Qiskit Runtime job can be over-estimated by up to eight hours, the maximum allowed Qiskit Runtime job duration for Premium Plan users.
 
-The job’s position in the queue is listed in the **Queue position** column on the [Jobs page](https://quantum-computing.ibm.com/jobs).
```

Comment:
Bad

Reply:
Wow, I'm so sorry, I'm new to this, will take care of it